server/config.go

```diff
@@ -16,27 +17,28 @@ const DefaultJSONRPCPort int = 8545
 type Config struct {
 	Chain *chain.Chain
 
-	JSONRPCAddr *net.TCPAddr
-	GRPCAddr    *net.TCPAddr
-	LibP2PAddr  *net.TCPAddr
-	Telemetry   *Telemetry
-	Network     *network.Config
-	DataDir     string
-	Seal        bool
-	Locals      []types.Address
-	NoLocals    bool
-	PriceLimit  uint64
-	MaxSlots    uint64
+	JSONRPCAddr    *net.TCPAddr
+	GRPCAddr       *net.TCPAddr
+	LibP2PAddr     *net.TCPAddr
+	Telemetry      *Telemetry
+	Network        *network.Config
+	DataDir        string
+	Seal           bool
+	Locals         []types.Address
+	NoLocals       bool
+	PriceLimit     uint64
+	MaxSlots       uint64
 	SecretsManager *secrets.SecretsManagerConfig
+	HandlersConfig *jsonrpc.Handlers
```

Comment:
I think at this point it is not scalable to add more fields into the base Config struct. Instead, create a JsonRPC config struct that holds all the jsonrpc configuration: bind addr and handlers. Note that this is what we do in Telemetry.